### DISCUSSION TITLE
React Example (documentProps not working)

### DISCUSSION THREAD

**Discussion by Niocas** [vikejs/vike#973] [Discussion]
The documentProps are not working for this example. The documentProps will always be undefined like this and show the default title.

![image](https://github.com/brillout/vite-plugin-ssr/assets/75745964/ab9edb9b-e8e9-4c18-af84-c9f879c8fac1)

**Comment by brillout** [comment]
It's possible, been a while I didn't touch that part of the example. PR welcome to fix the example.
